cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2800,6 +2800,7 @@ display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m Workload IN (OneDrive, SharePoint) Operation IN ("SharingSet", "AccessRequestApproved", "SharingRestored", "AnonymousLinkCreated", "SharingInvitationCreated", "SharingInvitationUpdated", "AddedToSecureLink", "SharingPolicyChanged") NOT TargetUserOrGroupType IN ("Member", "SharePointGroup") \
+|`cs_o365_excluding_microsoft_teams_filter` \
```

Comment:
update the macro name: cs_o365_file_sharing_to_external_user_internal_filter

and add it to macros.conf